pkg/kubectl/cmd/wait/wait.go

```diff
@@ -93,7 +93,7 @@ func NewWaitFlags(restClientGetter genericclioptions.RESTClientGetter, streams g
 			WithFieldSelector("").
 			WithAll(false).
 			WithAllNamespaces(false).
-			WithAll(false).
+			WithLocal(false).
```

Comment:
Yeah, I don't think this really makes a difference since the variable would be initialized to false anyway. Expliciit initialization doesn't hurt.

Reply:
@apelisse The `--local` flag of `kubectl wait` appeared in v1.16.0 due to initializing the variable.

```
$ kubectl version --client
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
$ kubectl wait -h | grep local
	--local=false: If true, annotation will NOT contact api-server but run locally.
```

How does the flag work in `kubectl wait`? If it have no effects, the variable should not be initialized.